video/out/w32_common.c

```diff
@@ -1624,6 +1650,9 @@ static void *gui_thread(void *ptr)
     update_dark_mode(w32);
     if (w32->opts->window_affinity)
         update_affinity(w32);
+#if __MINGW64_VERSION_MAJOR >= 12
+    update_backdrop(w32, true);
```

Comment:
This is just a more general thing but it would be better imo opinion if this function is always exposed. Instead, the guarding should be done so that the enum symbols are hidden and that the function is simply a no-op if you don't have it.